packages/sdk/lib/base.ts

```diff
@@ -439,15 +439,14 @@ class Itly {
   }
 
   private mergeContext(event: Event, context?: Properties): Event {
-    return context
-      ? {
-        ...event,
-        properties: {
-          ...context,
-          ...event.properties,
-        },
-      }
-      : event;
+    if (!context) {
```

Comment:
Nit picking here but feel it feel this could be a little more concise
`
return context
 ? Object.assign(Object.create(Object.getPrototypeOf(event)), event, {
   properties: { ...context, ...event.properties },
 })
 : event;
`

Reply:
Good catch, totally agree.